tendrl/ceph_integration/manager/utils.py

```diff
@@ -4,9 +4,9 @@
 import subprocess
 
 LOG = logging.getLogger(__name__)
-TENDRL_CONTEXT = "/etc/tendrl/ceph-integration/tendrl_context"
+TENDRL_CONTEXT = "/etc/tendrl/gluster-integration/integration_id"
 FSID = "/etc/tendrl/ceph-integration/fsid"
-NODE_CONTEXT = "/etc/tendrl/node_agent/node_context"
+NODE_CONTEXT = "/etc/tendrl/node-agent/NodeContext"
```

Comment:
Why are we still reading writing random files and not via object namespace?  The namespace way is pretty clear in node-agent and even in gluster-integration 

@brainfunked (please flag any such file read/writes or etcd read writes in any PR)